toolchain/lower/function_context.h

```diff
@@ -126,6 +139,47 @@ class FunctionContext {
   auto FinishInit(SemIR::TypeId type_id, SemIR::InstId dest_id,
                   SemIR::InstId source_id) -> void;
 
+  auto AddCallToCurrentFingerprint(SemIR::FunctionId function_id,
```

Comment:
There had been some discussion of trying to determine when the same entry came up repeated times, and outputting something different to indicate the repeat (rather than recalculating whatever you're using for hash). Have you thought about how that'd look?

Reply:
I haven't touched on this, future work! =]

Reply:
Is it worth adding some kind of TODO for that?